pset-iterators.js

```diff
@@ -66,6 +75,15 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter((president) => {
+  // console.log(president)
+  return president.president.includes("James")
```

Comment:
Great use of the filter iteration method combined with .includes to run your conditional!